mlflow/server/js/src/shared/web-shared/genai-traces-table/hooks/useMlflowTraces.tsx

```diff
@@ -640,6 +644,32 @@ const createMlflowSearchFilter = (
         case TracesTableColumnGroup.EXPECTATION:
           filter.push(`expectation.\`${networkFilter.key}\` ${networkFilter.operator} '${networkFilter.value}'`);
           break;
+        case SPAN_NAME_COLUMN_ID:
+          if (networkFilter.operator === '=') {
+            filter.push(`span.name ILIKE '${networkFilter.value}'`);
```

Comment:
should this be `=`? or it's intentionally ILIKE for usability?

```suggestion
            filter.push(`span.name = '${networkFilter.value}'`);
```